src/main/resources/default/assets/tycho/scripts/form.js.pasta

```diff
@@ -25,46 +25,56 @@ sirius.ready(function () {
         });
     });
 
-    const _modalElement = document.getElementById('link-confirm-modal');
+    const _modalElement = document.getElementById('link-confirm-modal-js');
     const _confirmForm = _modalElement.querySelector(".confirm-form-js");
     const _submitBtn = _modalElement.querySelector("button[type='submit']");
-    _submitBtn.addEventListener('click', function () {
+    _submitBtn.addEventListener('click', () => {
         sirius.requestSubmitForm(_confirmForm);
     });
 
-    document.querySelectorAll('.confirm-link-js').forEach(function (_node) {
-        _node.addEventListener('click', function (event) {
+    const modal = new bootstrap.Modal(_modalElement, {
+        keyboard: true,
+    });
+
+    document.querySelectorAll('.confirm-link-js').forEach(_node => {
+        _node.addEventListener('click', event => {
             try {
                 _confirmForm.setAttribute('action', _node.getAttribute('href'));
 
-                $('#link-confirm-modal').modal({
-                    keyboard: true
-                }).on('shown', function () {
-                    _modalElement.querySelector('.btn-close').focus();
-                }).attr('tabindex', -1);
-            } catch (e) {
-                console.log(e);
+                _modalElement.tabIndex = -1;
+                _modalElement.addEventListener('shown.bs.modal', () => {
+                    const _cancelButton = _modalElement.querySelector('.btn-cancel-js');
+                    if (_cancelButton != null) {
+                        _cancelButton.focus();
+                    }
+                }, {
+                    once: true,
+                });
+
+                modal.show(null);
```

Comment:
```suggestion
                modal.show();
```
Or does this not work? normally a not given parameter in JS should be undefined in the called method, so most probably the behaviour is the same

Reply:
I agree that yours is the better code, and it works without an issue…

However, (faulty?) Code Inspection wants to have the superfluous `null` there for some reason. I decided to put it in to suppress a pointless warning.

![Screenshot 2024-02-09 at 18 22 37](https://github.com/scireum/sirius-web/assets/36069303/df15e47f-4406-4649-a665-97685a65016d)

In the light of this, which version do you prefer?